specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2018-10-01/imds.json

```diff
@@ -0,0 +1,731 @@
+{   
+    "swagger": "2.0",
+    "info": {
+        "title": "InstanceMetadataClient",
+        "description": "The Azure Instance Metadata Client",
+        "version": "2018-10-01"
+    },
+    "host": "169.254.169.254",
+    "basePath": "/metadata",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "security": [
+        {
+            "azure_auth": [
+                "user_impersonation"
+            ]
+        }
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/instance": {
+            "get": {
+                "operationId": "getInstance",
+                "description": "Get Instance Metadata for the Virtual Machine.",
+                "parameters": [
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "required": true,
+                        "type": "string",
+                        "description": "This is the version of API to invoke."
+                    },
+                    {
```

Comment:
names of the parameter should be camelCase

Reply:
Move api-version to "parameters" section, under "ApiVersionParameter"